terraform/main.tf

```diff
@@ -78,16 +117,18 @@ resource "google_compute_instance" "gateway_vm" {
 
   provisioner "remote-exec" {
     scripts = [
+      "./scripts/setup_ca.sh",
       "./scripts/download_node_exporter.sh",
-      "./scripts/run_node_exporter.sh"
+      "./scripts/run_node_exporter.sh",
+      "./scripts/run_docker_container.sh"
```

Comment:
as per Janos's suggestion, running container inside gateway-vm to run sshcontainer